councilmatic_core/models.py

```diff
@@ -24,21 +24,29 @@
 class CastToDateTimeMixin:
 
     @classmethod
-    def cast_to_datetime(cls, field):
+    def _cast(cls, field, field_type):
         """
-        Cast a given field from a CharField to a DateTimeField, converting empty
-        strings to NULL in the process. Useful for CharFields that store timestamps
-        as strings.
+        Cast a given field from a CharField to a DateField or a DateTimeField,
+        converting empty strings to NULL in the process. Useful for CharFields
+        that store timestamps as strings.
         """
         return Cast(
             Case(
                 When(**{field: '', 'then': None}),
                 default=field,
                 output_field=models.CharField()
             ),
-            models.DateTimeField()
+            field_type
         )
 
+    @classmethod
+    def cast_to_date(cls, field):
+        return cls._cast(field, models.DateField())
+
+    @classmethod
+    def cast_to_datetime(cls, field):
```

Comment:
do we need this path around?

Reply:
Good question! We're still using it for [event start dates](https://github.com/datamade/django-councilmatic/blob/02bea3f620b92eb284f3f8c13f327daa69dfe5be/councilmatic_core/models.py#L315), as well as [membership start and end dates](https://github.com/datamade/django-councilmatic/blob/02bea3f620b92eb284f3f8c13f327daa69dfe5be/councilmatic_core/models.py#L273). We'll eventually transition membership start/end dates, but events actually come with a date time string.

Reply:
ty.